File: sdk/prismer-cloud/golang/aip.go
package prismer

// Prismer SDK — AIP Identity (Platform Integration)
//
// Go SDK re-exports from the standalone aip-sdk-go module.
// Import the standalone SDK directly for pure AIP usage:
//
//	import aip "github.com/nicepkg/aip-sdk-go"
//	id, _ := aip.NewAIPIdentity()
//
// The prismer-sdk-go module adds platform-specific wrappers (v1.7.4 planned):
//
//	import prismer "github.com/nicepkg/prismer-sdk-go"
//	agent, _ := prismer.RegisterAIPAgent(client, apiKey)

// NOTE: In Go, re-exporting from another module requires explicit type aliases.
// Until the standalone aip-sdk-go is published, the platform SDK embeds the
// implementation directly. After publishing, this file will import from
// github.com/nicepkg/aip-sdk-go and provide thin wrappers.
//
// For now, users should import the standalone SDK directly from sdk/aip/golang/.
// See sdk/aip/golang/ for the full AIP SDK implementation.

File: sdk/prismer-cloud/rust/src/aip.rs
//! Prismer SDK — AIP Identity (Platform Integration)
//!
//! Re-exports from the standalone `aip-sdk` crate.
//! Add `aip-sdk` to your Cargo.toml dependencies:
//!
//! ```toml
//! [dependencies]
//! aip-sdk = "1.7.3"
//! prismer-sdk = "1.7.3"
//! ```
//!
//! For standalone AIP usage without Prismer:
//! ```rust
//! use aip_sdk::AIPIdentity;
//! let id = AIPIdentity::create();
//! ```
//!
//! Platform integration (v1.7.4 planned):
//! ```rust
//! use prismer_sdk::aip::PrismerAIPAgent;
//! let agent = PrismerAIPAgent::register(&client, api_key).await;
//! ```

// NOTE: After aip-sdk crate is published to crates.io, this module will
// re-export from it via `pub use aip_sdk::*;`. Until then, users should
// depend on the standalone crate directly from sdk/aip/rust/.
